lib/api/objectPut.js

```diff
@@ -119,7 +120,7 @@ function _storeIt(bucketName, objectKey, objMD, authInfo, canonicalID,
                                 newByteLength: size,
                                 oldByteLength: prevContentLen,
                             });
-                            return callback(null, contentMD5, prevContentLen);
```

Comment:
UtapiClient uses the value of `prevContentLen` to calculate the storage utilized metric if the operation is overwriting an object. Should be `null` if not overwriting.